src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.html

```diff
@@ -88,17 +87,64 @@ <h2 *ngIf="loggedIn" id="wi-detail-title-click" class="detail-title truncate">{{
           </div>
         </div>
       </div>
-      <div class="width100 pull-left paddingL20 paddingR20">
-          <p class="pull-left margin0 paddingT5" id="wi-detail-assignee-title">
-              Assignee
-          </p>
-          <div class="pull-left detail-assignee-avatar">
-              <span class="pficon-user pull-left" id="wi-detail-assignee"></span>
-          </div>                
+      <div class="width100 pull-left paddingL20 paddingR20 detail-assignee-wrap">
+          <span
+            *ngIf="!assignedUser"
+            (click)="activeSearchAssignee()" 
+            class="pull-left fa fa-user-plus user-assign-icon">
+          </span>
+          <img 
+            *ngIf="assignedUser"
+            (click)="activeSearchAssignee()"
+            class="pull-left user-assign-avatar"
+            src="{{assignedUser.imageURL + '&s=20'}}" 
+            onError="this.src='https://avatars0.githubusercontent.com/u/563119?v=3&s=20'" />
+          
+          
+          <div class="pull-left detail-assignee-name">
+              <span 
+                *ngIf="!searchAssignee && !assignedUser" 
+                (click)="activeSearchAssignee()" 
+                class="placeholder clickable"> Unassigned </span>
+              
+              <span 
+                *ngIf="!searchAssignee && assignedUser" 
+                (click)="activeSearchAssignee()" 
+                class="clickable"> {{assignedUser.fullName}} </span>
+
+              <div *ngIf="searchAssignee" class="user-search" >
+                <div class="list-container">
+                  <span class="fa fa-search user-search-icon"> </span>                  
+                  <input #userSearch (keyup)="filterUser($event)" />
+                  <ul class="user-list" #userList>
+                    <li 
+                      class="item-li" 
+                      *ngFor="let user of filteredUsers;" 
+                      [attr.data-value]="user.id"
+                      (click)="assignUser(user.id)">
+                        <!-- <span class="pull-left fa fa-user user-assign-icon active"></span> -->
+                        <img 
+                          class="pull-left user-assign-avatar" 
+                          src="{{user.attributes.imageURL + '&s=20'}}" 
+                          onError="this.src='https://avatars0.githubusercontent.com/u/563119?v=3&s=20'"/>
+                        <span [innerHTML]="user.attributes.fullName | almSearchHighlight: userSearch.value"></span>
+                    </li>
+                  </ul>
+                  <ul>
+                    <li class="action-li">
+                        <span class="action-item" (click)="unassignUser()">Unassign</span>
+                        <span class="action-item" (click)="cancelAssignment()">Cancel</span>
```

Comment:
Just in case you open the list and then decide not to assign anyone or update. You can either hit ESC or click on cancel to close the list.